src/lib.rs

```diff
@@ -179,12 +248,43 @@ pub fn make_game() -> MakeGameResult {
             height,
         }) => MakeGameResult {
             game: Some(Game {
-                x: 0.0,
-                y: 0.0,
                 speed: 2.0,
                 context,
                 width,
                 height,
+                local_player_color: Some(Color::random()),
+                players: vec![
+                    Player {
+                        color: Color::Red,
+                        x: 275.0 + (100.0 * (1.0 / 6.0 * 2.0 * f64::consts::PI).sin()),
+                        y: 275.0 + (100.0 * (1.0 / 6.0 * 2.0 * f64::consts::PI).cos()),
+                    },
```

Comment:
Could these players have been instantiated in a loop somehow?

Reply:
+1, done